### PR TITLE
US-001 - Addition of java files and html and js files

### PR DESCRIPTION
**Pull request by John-Neill-Kainos**
THese java files allows a connection between the mysql and the front end web using spring boot.

Changes made to html files to show proof of concept for working tables

### CODE REVIEW

**Review by RyanCunning1 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,5 @@
 
 **/*.iml
 
-/.DS_STORE
+/application/.DS_STORE
```
Comment (jkelso662) [inline .gitignore:9]: Not going to block the PR but these two can be combined. Have a look at https://git-scm.com/docs/gitignore

File: Views/JobsList.html [hunk 2]
```diff
@@ -7,6 +7,8 @@
     <link rel="preload" href="https://www.kainos.com/Static/dist/styles.css">
     <meta charset="utf-8">
     <title>Jobs List</title>
+    
+
```
Comment (jkelso662) [inline Views/JobsList.html:11]: Not a fan of all the blank space in HTML. Would remove this throughout. Try using the IDE to auto-format maybe?

File: Views/JobsList.html [hunk 3]
```diff
@@ -344,8 +340,19 @@ <h3 class="mega-menu__secondary-column-title">Industries</h3>
...
                 <div class="row">
-                    <div class="col-12 col-md-8 mx-auto">
-                        <h2 class="text-center-block__title">List of Job Roles</h2>
+                <h2 class="text-center-block__title">List of Job Roles</h2>
+                <br>
+                <br>
+                <br>
+                <div class="text-center-block__title" id = "Job Roles">
+
+                        <script> console.log("Hello") </script>
```
Comment (jkelso662) [inline Views/JobsList.html:349]: Surely this shouldn't be in master?

File: application/apiService/.gitignore [hunk 4]
```diff
@@ -0,0 +1,38 @@
+HELP.md
```
Comment (jkelso662) [inline application/apiService/.gitignore:1]: How come you have two gitignores?

File: Views/JobsList.html (hunk 3 above)
Comment (John-Neill-Kainos) [reply Views/JobsList.html:349]: Yep correct, i just forgot about these

File: application/apiService/.gitignore (hunk 4 above)
Comment (John-Neill-Kainos) [reply application/apiService/.gitignore:1]: There is a git ignore in the java project and then one was automatically created when Liam created the code for the front end

File: application/apiService/.mvn/wrapper/MavenWrapperDownloader.java [hunk 5]
```diff
@@ -0,0 +1,117 @@
+/*
```
Comment (jkelso662) [inline application/apiService/.mvn/wrapper/MavenWrapperDownloader.java:1]: Not sure about things in application/apiService/.mvn/ being added. If a folder is hidden (I.E. has a dot before the name) it is possible that it shouldn't be committed. Have a look at something like this to see what should be committed https://stackoverflow.com/questions/51792830/what-files-in-a-maven-project-should-be-committed-to-git

File: application/apiService/src/main/java/com/kainos/ea/apiService/controllers/apiService.java [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+            System.out.println(e);
+        }
+
+    }
+
+    @GetMapping("/jobslist")
+    public ResponseEntity<String> get() {
+        JSONArray queryInformation = null;
+
+
```
Comment (jkelso662) [inline application/apiService/src/main/java/com/kainos/ea/apiService/controllers/apiService.java:59]: Not a huge fan of all this white space but if it's agreed as the standard across the team then work away.

File: application/apiService/.gitignore (hunk 4 above)
Comment (jkelso662) [reply application/apiService/.gitignore:1]: Was there a `git clone ...` as part of that new front end code? Should only ever be one at the root of the git repo.

Comment (John-Neill-Kainos) [reply application/apiService/.gitignore:1]: okay should i then delete the one in the java project

**Review by christina-mcmullan — APPROVED** [review]

**Pull request merged**